Dockerfile

```diff
@@ -27,12 +30,14 @@ RUN cd dist && \
 FROM builder as google-authenticator-libpam-builder
 
 ARG AUTHENTICATOR_LIBPAM_VERSION=1.05
-RUN git clone --branch ${AUTHENTICATOR_LIBPAM_VERSION} --single-branch https://github.com/google/google-authenticator-libpam dist
+RUN git clone --branch ${AUTHENTICATOR_LIBPAM_VERSION} --single-branch https://github.com/google/google-authenticator-libpam src
 
-RUN cd dist && \
+RUN cd src && \
```

Comment:
Question: do we gain anything by chaining commands with `&&` in a multi-stage build where the build step is not shipped in the final image?

Reply:
just looks cleaner and reduces cached layers (disk space)